worker/include/RTC/TransportCongestionControlServer.hpp

```diff
@@ -59,6 +61,13 @@ namespace RTC
 		void SendTransportCcFeedback();
 		void MaySendLimitationRembFeedback();
 		void UpdatePacketLoss(double packetLoss);
+		void OnPacketArrival(uint16_t sequence_number, uint64_t arrival_time);
+		void SendPeriodicFeedbacks();
+		int64_t BuildFeedbackPacket(
+		  int64_t base_sequence_number,
+		  std::map<int64_t, uint64_t>::const_iterator begin_iterator, // |begin_iterator| is inclusive.
+		  std::map<int64_t, uint64_t>::const_iterator end_iterator    // |end_iterator| is exclusive.
```

Comment:
We don't use such a `|xxxx|` anywhere. Also, variables in mediasoup must be in lowcase.

```suggestion
		  std::map<int64_t, uint64_t>::const_iterator beginIterator, // It is inclusive.
		  std::map<int64_t, uint64_t>::const_iterator endIterator    // It is exclusive.
```